### PR TITLE
perf: Incorrect/Missing refs on types (#902)

### PR DESCRIPTION
**Pull request by Waqar144**

### CODE REVIEW

**Comment by egraether** [comment]
Thank you taking care of this!

Sorry, but you will need to rebase this to master again, since I committed an old branch yesterday.

Why does this contain changes that only affect formatting? We use `clang-format` and I don't want to cause back-and-forth changes due to competing formatting rules.

**Comment by Waqar144** [comment]
I ran clang-format on the whole folder, I guess that's why. Sorry 😬

**Comment by egraether** [comment]
If you used clang-format, then it's ok and I will accept the changes. Some rules seem to need a couple iterations until stable. We just recently started using it.

**Comment by Waqar144** [comment]
Rebased!

File: src/lib_gui/qt/project_wizard/QtProjectWizard.cpp [hunk 1]
```diff
@@ -1012,12 +1013,12 @@ void QtProjectWizard::createProject()
 	{
 		MessageStatus(L"Unable to save project to location: " + path.wstr()).dispatch();
 
-		QMessageBox msgBox(this);
```
Comment (egraether) [inline src/lib_gui/qt/project_wizard/QtProjectWizard.cpp]: Please add the `(this)` again. Got probably lost during rebase.

Comment (Waqar144) [reply src/lib_gui/qt/project_wizard/QtProjectWizard.cpp]: ah okay.

**Comment by Waqar144** [comment]
> Thanks! For the future: Please always create new commits to address a review. That way the reviewer only needs to look at the new commits and not the whole PR again. We use 'Squash and Merge' for pull requests, so every PR ends up being a single commit anyways.

Alright. I will keep that in mind :)

**Pull request merged**